cesql/cesql_tck/README.md

```diff
@@ -25,3 +25,4 @@ The `error` values could be any of the following:
 * `cast`: Casting error
 * `missingFunction`: Addressed a missing function
 * `functionEvaluation`: Error while evaluating a function
+* `missingAttribute`: Error due to a missing attribute
```

Comment:
I think you added a generic error message to the spec - should we test for that too? It might end up being more of an impl-specific error case. If it's too hard to put into this generic tck stuff then I'm ok with skipping it. Was just curious.

Reply:
I think we could mention it in the `README` for completeness, but I can't think of any use cases where we would actually use it